packages/react-aria-components/src/FileTrigger.tsx

```diff
@@ -52,7 +52,14 @@ function FileTrigger(props: FileTriggerProps, ref: ForwardedRef<HTMLDivElement>)
       <PressResponder onPress={() => inputRef.current?.click()}>
         {children}
       </PressResponder>
-      <Input type="file" style={{display: 'none'}} ref={inputRef} accept={acceptedFileTypes?.toString()} onChange={onChange} capture={defaultCamera} multiple={allowsMultiple} /> 
+      <Input 
+        type="file" 
+        style={{display: 'none'}} 
```

Comment:
One last nit, for consistency, we can wrap this `Input` in a `VisuallyHidden` instead of applying this style